src/libcore/iter/adapters/mod.rs

```diff
@@ -253,6 +323,7 @@ where
 }
 
 #[doc(hidden)]
+#[unstable(issue = "none", feature = "std_internals")]
```

Comment:
Rebase problem?

Reply:
no, TrustedRandomAccess is public now (but hidden and unstable) so that libcore can share it with liballoc, this lead to the compiler complaining about these attributes